.github/workflows/run-tests.yml

```diff
@@ -61,7 +63,6 @@ jobs:
 
       - name: Install dependencies
         run: |
-          composer require "laravel/framework:${{ matrix.laravel }}" "orchestra/testbench:${{ matrix.testbench }}" --no-interaction --no-update
```

Comment:
Why is this line being removed? This removes the feature of the matrix?

Reply:
With it composer fails to install dependencies, see: https://github.com/gosuperscript/zero-downtime-event-replays/actions/runs/10323133366/job/28579861212

I aligned it with other packages that are just using update with `stability`, ie https://github.com/morrislaptop/laravel-popo-caster/pull/8/files#diff-7314d0ebbd2e9537ae4889316745b4fd2fa43cb86275c9caae18a86ba228b642R46 and you can see in CI run: https://github.com/gosuperscript/zero-downtime-event-replays/actions/runs/10323510087/job/28581028976 that it does install lowest / highest version for each package

Reply:
I guess we won't be explicitly using testbench on given version, but that will be the same in other packages. Do you want to align other packages with this one?